tests/integration/test_bayesian_optimization.py

```diff
@@ -113,6 +114,14 @@ def GPR_OPTIMIZER_PARAMS() -> Tuple[str, List[ParameterSet]]:
                 ),
                 id="MinValueEntropySearch",
             ),
+            pytest.param(
```

Comment:
Have you checked that the slow tests pass too? You can do this locally by running `tox -e tests -- --runslow yes` or (just adding `--runslow yes` when running `test_bayesian_optimizer_with_gpr_finds_minima_of_scaled_branin`). Alternatively you can kick off a manual `develop_checks` github run on your branch ­and check that alltests_latest passes (alltests is currently crashing for an unrelated issue).

Reply:
Yes, I'll check these tests are passing locally. Are the tests not run remotely whenever one pushes to the branch anyway though?

Reply:
Just ran the slow tests locally and they pass :)